indexing-service/src/test/java/org/apache/druid/indexing/common/task/TaskSerdeTest.java

```diff
@@ -116,23 +116,31 @@ public void testIndexTaskTuningConfigTargetPartitionSizeOrNumShards() throws Exc
         IndexTask.IndexTuningConfig.class
     );
 
-    Assert.assertEquals(10, (int) tuningConfig.getTargetPartitionSize());
+    Assert.assertEquals(10, (int) tuningConfig.getMaxRowsPerSegment());
```

Comment:
nit: please add a backwards compatibility test if not already present.

Reply:
Thanks. I added a test when both maxRowsPerSegment and targetPartitionSize don't exist. We already have a test for when only targetPartitionSize is provided [here](https://github.com/apache/incubator-druid/pull/6780/files#diff-be6c09b426aded436da657df0be06cffR154).